src/renderer/components/+custom-resources/crd-details.tsx

```diff
@@ -124,7 +124,7 @@ export class CRDDetails extends React.Component<Props> {
         <>
           <DrawerTitle title="Validation"/>
           <AceEditor
-            mode="json"
+            mode="yaml"
```

Comment:
This caused a crash because we are not loading any `json` mode for the `AceEditor`. This was the only place that `yaml` wasn't used and `yaml` is a superclass of `json` so this should be fine (it looked fine to me).

Reply:
Add `require("ace-builds/src-noconflict/mode-json");` in constructor in `ace-editor.tsx` to fix this.

Reply:
Sure, but crd details probably could be `yaml` too. So I think this is the better fix.